### PR TITLE
Remove unused strings from translation files

### PR DESCRIPTION
**Pull request by gandaro**

### CODE REVIEW

File: config/locales/diaspora/en.yml [hunk 1]
```diff
@@ -1315,7 +1311,6 @@ en:
       hide_and_mute: "Hide and mute post"
       like: "Like"
       unlike: "Unlike"
-      dislike: "Dislike"
```
Comment (gandaro) [inline config/locales/diaspora/en.yml]: I wonder if this string is really not needed anymore… I couldn’t find it used anywhere, but there are references to dislikes in *lib/diaspora/likeable.rb* and *spec/controllers/likes_controller_spec.rb*.

**Comment by gandaro** [comment]
Please check my comment on the diff before merging.

**Comment by denschub** [comment]
Have you checked if there are any other unused keys?

**Comment by denschub** [comment]
#6901.

**Pull request closed without merging**